configs/api/prod.yaml

```diff
@@ -1,3 +1,12 @@
+# Angus: Remember that in prod, we typically want to inject sensitive environment variables like connection strings,
```

Comment:
Ok now I uderstand more in depth what you mean by injecting the general config in the environment. Because yes, as there are sensitive information in the config file, we can't store it in the repo. I really like the last option too, I will implement it in the template. However, I'm intrigued to hear more of your thoughts on this matter.
In a more general case like in a company, what do you think about this? Does the yaml file is a good option to store application configuration with non sensitive information, like application address, port etc...?